data/mainnet/tokens.js

```diff
@@ -45,6 +47,482 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['FARM', 'WETH'],
   },
+  iFARM_arbitrum: {
+    chain: CHAINS_ID.ARBITRUM_ONE,
+    decimals: '18',
+    tokenAddress: addresses.ARBITRUM_ONE.iFARM,
+    displayName: 'aiFARM',
+    vaultAddress: null,
+    priceFunction: { type: GET_PRICE_TYPES.F_TOKEN, params: [addresses.iFARM, '18'] },
+  },
+  Crv2Pool_arbitrum: {
+    chain: CHAINS_ID.ARBITRUM_ONE,
+    category: VAULT_CATEGORIES_IDS.CURVE,
+    logoUrl: './icons/curve-crypto-icons-stack-2-arbitrum.svg#2pool',
+    decimals: '18',
+    tokenAddress: addresses.ARBITRUM_ONE.Crv2Pool_Strategy.Underlying,
+    displayName: 'CRV:2Pool',
```

Comment:
Also make this consistent with our other vaults. displayName should be the tokens in the LP, so `USDC-USDT`, subLabel: `Curve`. Same comment for all other vaults